packages/fury-adapter-oas3-parser/lib/parser/oas/parseOauthFlowsObject.js

```diff
@@ -0,0 +1,88 @@
+const R = require('ramda');
+const {
+  isAnnotation, isExtension, hasKey,
+} = require('../../predicates');
+const {
+  createWarning,
+  createInvalidMemberWarning,
+} = require('../annotations');
+const pipeParseResult = require('../../pipeParseResult');
+const parseObject = require('../parseObject');
+const parseOauthFlowObject = require('./parseOauthFlowObject');
+
+const name = 'Oauth Flows Object';
+
+const validFlow = R.anyPass(R.map(hasKey, ['implicit', 'password', 'clientCredentials', 'authorizationCode']));
```

Comment:
We're prefixing these types of functions (predicates) with `is` elsewhere. Let's do same here for consistency?

- isString, isAnnotation, isExtension, etc

```
lib/parser/oas/parseParameterObject.js:const isValidInValue = R.anyPass([
lib/parser/oas/parseSchemaObject.js:const isValidType = R.anyPass(R.map(hasValue, types));
lib/parser/oas/parseSecuritySchemeObject.js:const isValidTypeValue = R.anyPass([
lib/parser/oas/parseSecuritySchemeObject.js:const isValidInValue = R.anyPass([
```